Airports/EGCC/SMR/Geo.txt

```diff
@@ -1243,15 +1243,25 @@ N053.21.59.434 W002.17.20.195 N053.21.59.748 W002.17.20.357 smrCCtaxiway
 N053.21.59.748 W002.17.20.357 N053.21.59.983 W002.17.20.635 smrCCtaxiway
 N053.21.59.983 W002.17.20.635 N053.22.02.593 W002.17.25.058 smrCCtaxiway
 N053.22.02.593 W002.17.25.058 N053.22.02.750 W002.17.25.303 smrCCtaxiway
-N053.22.02.750 W002.17.25.303 N053.22.02.957 W002.17.25.491 smrCCtaxiway
-N053.22.02.957 W002.17.25.491 N053.22.03.181 W002.17.25.592 smrCCtaxiway
-N053.22.03.181 W002.17.25.592 N053.22.03.395 W002.17.25.606 smrCCtaxiway
-N053.22.03.395 W002.17.25.606 N053.22.03.584 W002.17.25.549 smrCCtaxiway
-N053.22.03.584 W002.17.25.549 N053.22.03.817 W002.17.25.419 smrCCtaxiway
-N053.22.03.817 W002.17.25.419 N053.22.05.506 W002.17.24.415 smrCCtaxiway
-N053.22.05.506 W002.17.24.415 N053.22.05.776 W002.17.24.181 smrCCtaxiway
-N053.22.05.776 W002.17.24.181 N053.22.05.977 W002.17.23.889 smrCCtaxiway
-N053.22.05.977 W002.17.23.889 N053.22.13.000 W002.17.12.236 smrCCtaxiway
+N053.22.02.750 W002.17.25.303 N053.22.02.957 W002.17.25.491 smrCCOrange
```

Comment:
Is it worth splitting this off into a separate `;Taxiway_Z_Orange` section? I guess if you haven't editted the .kmz as such leaving it as a single item kind of makes sense, it just seems a little unusual to have a line change colour part way along it with no comment explaining why.